mycore-webtools/src/main/resources/META-INF/web-fragment.xml

```diff
@@ -0,0 +1,18 @@
+<web-fragment version="6.0" xmlns="https://jakarta.ee/xml/ns/jakartaee"
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-fragment_6_0.xsd"
+              metadata-complete="true">
+
+    <name>mycore_web_tools</name>
+
+
+    <servlet>
+        <servlet-name>MCRPropertyHelperContentServlet</servlet-name>
+        <servlet-class>org.mycore.webtools.properties.MCRPropertyHelperContentServlet</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>MCRPropertyHelperContentServlet</servlet-name>
+        <url-pattern>/servlets/PropertyHelperContentServlet</url-pattern>
+    </servlet-mapping>
```

Comment:
Should this be always enabled? When the developer commands `show property`, ... were added, we decided that property-revealing (and therefore potentially secret-revealing) features should be opt-in.

Reply:
I added the property `MCR.Servlet.MCRPropertyHelperContentServlet.Disabled=true` to disable it by default.